provisioner/converge/provisioner.go

```diff
@@ -0,0 +1,262 @@
+// This package implements a provisioner for Packer that executes
+// Converge to provision a remote machine
+
+package converge
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"log"
+	"net/http"
+
+	"strings"
+
+	"encoding/json"
+
+	"regexp"
+
+	"github.com/mitchellh/packer/common"
+	"github.com/mitchellh/packer/helper/config"
+	"github.com/mitchellh/packer/packer"
+	"github.com/mitchellh/packer/template/interpolate"
+)
+
+var versionRegex = regexp.MustCompile(`^[\.\-\da-zA-Z]*$`)
+
+// Config for Converge provisioner
+type Config struct {
+	common.PackerConfig `mapstructure:",squash"`
+
+	// Bootstrapping
+	Bootstrap bool   `mapstructure:"bootstrap"`
+	Version   string `mapstructure:"version"`
```

Comment:
I'm sceptical about the usefulness of this? Why not just assume it's on the path if `"bootstrap": false` and use latest in case of `"bootstrap": true`